src/snapshot_middleware/mod.rs

```diff
@@ -14,7 +14,7 @@ mod project;
 mod rbxm;
 mod rbxmx;
 mod txt;
-mod util;
+pub mod util;
```

Comment:
I don't think we should be exporting utilities out of this module. See my other comment on the code using this export.